benchmarks/histogram_bench.lua

```diff
@@ -0,0 +1,195 @@
+local metrics = require 'override.metrics'
```

Comment:
```suggestion
local metrics = require 'metrics'
```
already takes `override.metrics`.

Reply:
good point